app/views/follow_requests/create.js.erb

```diff
@@ -0,0 +1,2 @@
+let updated_follow_button = 
```

Comment:
This is an interesting syntax, I am curious as why you used this approach compared to the likes jquery

Reply:
Consider making changes to your unique id in the div in the _follow_unfollow.html.erb and editing your code in this file!